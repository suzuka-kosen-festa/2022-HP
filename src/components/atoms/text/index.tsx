import tw from "twin.macro";

const Text = tw.p`font-zen text-text text-xs sm:text-base select-none`;

export { Text };
