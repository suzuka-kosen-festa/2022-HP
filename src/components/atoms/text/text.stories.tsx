import type { ComponentStoryObj, ComponentMeta } from "@storybook/react";
import { Text } from ".";

type T = typeof Text;
type Story = ComponentStoryObj<T>;

const data = {
  sentence: "test",
};

export default { args: { children: data.sentence }, component: Text, title: "Text" } as ComponentMeta<T>;

export const Default: Story = {};
