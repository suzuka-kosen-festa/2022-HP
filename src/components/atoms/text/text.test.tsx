import { composeStories } from "@storybook/testing-react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import * as stories from "./text.stories";

const { Default } = composeStories(stories)

describe("(components) atom/text", () => {
  test("snap shot", () => {
    const { container } = render(<Default />)
    expect(container).toMatchSnapshot()
  })
})
