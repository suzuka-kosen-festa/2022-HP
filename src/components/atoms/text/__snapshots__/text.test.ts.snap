// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Text/Text Default 1`] = `
<span
  className="css-lz71gy-Text e8xjir80"
>
  This is test
</span>
`;
